sos/plugins/kubernetes.py

```diff
@@ -60,70 +50,73 @@ def setup(self):
         for svc in svcs:
             self.add_journal(units=svc)
 
-        # We can only grab kubectl output from the master
-        if self.check_is_master():
-            kube_cmd = "kubectl "
-            if path.exists('/etc/origin/master/admin.kubeconfig'):
-                kube_cmd += "--config=/etc/origin/master/admin.kubeconfig"
-
-            kube_get_cmd = "get -o json "
-            for subcmd in ['version', 'config view']:
-                self.add_cmd_output('%s %s' % (kube_cmd, subcmd))
-
-            # get all namespaces in use
-            kn = self.get_command_output('%s get namespaces' % kube_cmd)
-            knsps = [n.split()[0] for n in kn['output'].splitlines()[1:] if n]
-
-            resources = [
-                'limitrange',
-                'pods',
-                'pvc',
-                'rc',
-                'resourcequota',
-                'services'
-            ]
-
-            # nodes and pvs are not namespaced, must pull separately
-            self.add_cmd_output([
-                "{} get -o json nodes".format(kube_cmd),
-                "{} get -o json pv".format(kube_cmd)
-            ])
-            if self.get_option('all'):
-                for n in knsps:
-                    knsp = '--namespace=%s' % n
-                    k_cmd = '%s %s %s' % (kube_cmd, kube_get_cmd, knsp)
-
-                    self.add_cmd_output('%s events' % k_cmd)
+        kube_cmd = "kubectl "
+        kubeconfig_path = ""
+        if self.get_option('kubeconfig'):
+            kubeconfig_path = self.get_option('kubeconfig')
+        else:
+            kubeconfig_path = expanduser("~/.kube/config")
+        if kubeconfig_path != "" and path.exists(kubeconfig_path):
+            kube_cmd += "--kubeconfig=" + kubeconfig_path
```

Comment:
I suggest adding `+ " "` at the end. To have aligned kube_cmd with trailing space in either case (i.e. if someone will add another option in advance).

Or alternatively, set `kube_cmd = "kubectl"` and then `kube_cmd += " --kubeconfig="..`.

Reply:
Yes. Updated with your suggestion.